services/thumbnails/pkg/config/config.go

```diff
@@ -26,7 +26,7 @@ type Config struct {
 
 // FileSystemStorage defines the available filesystem storage configuration.
 type FileSystemStorage struct {
-	RootDirectory string `yaml:"root_directory" env:"THUMBNAILS_FILESYSTEMSTORAGE_ROOT" desc:"The directory where the filesystem storage will store the thumbnails."`
+	RootDirectory string `yaml:"root_directory" env:"THUMBNAILS_FILESYSTEMSTORAGE_ROOT" desc:"The directory where the filesystem storage will store the thumbnails. If not definied, the root directory defaults to $OCIS_BASE_DATA_PATH:/thumbnails."`
```

Comment:
```suggestion
	RootDirectory string `yaml:"root_directory" env:"THUMBNAILS_FILESYSTEMSTORAGE_ROOT" desc:"The directory where the filesystem storage will store the thumbnails. If not definied, the root directory derives from $OCIS_BASE_DATA_PATH:/thumbnails."`
```
Aligned to the wording from the PR where the other envs are corrected